digiwf-gateway/src/main/resources/application-local.yml

```diff
@@ -3,7 +3,7 @@ server:
 spring:
 
   # Define the local keycloak realm here
-  realm: <YOUR_LOCAL_REALM>
+  realm: P82
```

Comment:
Overwrite this setting in an env variable or in application-internal.properties?

Reply:
If you think this is kind of a secret I can revert that. But i dont think so.

Reply:
@lmoesle What do you think?